### ISSUE TITLE
K8s engine failed to start on macOS 10.15.6 

### ISSUE THREAD

**Issue by gengmao** [docker/for-mac#4831]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 5DD2EB55-FA3E-4288-B6DD-791ED8132DE5/20200804185151

### Expected behavior
Start Kubernetes engine

### Actual behavior
Failed to start. Reset/Uninstall didn't work.

### Information

  - macOS Version: 10.15.6 

### Diagnostic logs

```
Docker for Mac: version...
Version: 2.3.0.4 (46911) 
Channel: stable 
```

### Steps to reproduce the behavior

  1. upgrade docker for mac to latest stable (ver 2.3.0.4) or edge channels.

**Comment by chujanen** [comment]
I am also having this issue.

**Comment by williamli** [comment]
@gengmao @chujanen are you guys using a non-default location for your vm files?

**Comment by gengmao** [comment]
I was using default

**Comment by aselius** [comment]
i also have the same issue.

**Comment by raix** [comment]
Also experience this issue - any chance for a fix primo next week? (or is the solution to downgrade docker desktop?)

**Comment by dangthanh360** [comment]
am also having this issue too

**Comment by jasonaliyetti** [comment]
I hit this.  I did a complete uninstall/reinstall and it started working again.

**Comment by chujanen** [comment]
I ended up doing the same as @jasonaliyetti and it also started working.

**Comment by gengmao** [comment]
Installed supplemental update for macOS Catalina 10.15.6, uninstalled and reinstalled Docker edge, problem remains.

**Comment by cfcode** [comment]
Confirmed: Installed supplemental update for macOS Catalina 10.15.6, uninstalled and reinstalled Docker edge, fixed my problem.
Only the FROM image/OS in dockerfile rule is about 5 minutes loading. before update it download directly the OS image.

**Comment by AlexandreRoba** [comment]
Hi all, 

I had the same problem on  Mac OS catalina 10.15.6 and latest update.
I Reset,Clean,Reset, uninstall, reboot, reinstall, .... Tried a lot of different things nothing worked. Kubernetes was always failing to start...

I ran a diagnostics and went through the logs. i would see some error related to contexts...
I'm connecting to several cluster and have around 10 kubectl context on my machine.

I made sure to be on the Docker-desktop context and restarted docker desktop. Then Kubernetes started successfuly.

So make sure to be have the proper context set before starting it:
```
kubectl config use-context docker-desktop
```
 Hope this helps anyone.
